example/docker-compose.extra.yml

```diff
@@ -120,9 +125,9 @@ services:
         aliases:
          - ${QUICKSTATEMENTS_HOST}
```

Comment:
It is subtle, and worth noting here and perhaps somewhere in our docs. How this magic works is that we are making the `QUICKSTATEMENTS_HOST` (e.g. "quickstatements.local.gd") a Docker network alias for this service. This means that when inside the docker container when the Wikibase service goes to "http://quickstatements.local.gd" it will go to this container. Conversely on the host, when the nginx-proxy is running, it will route to this container because of the `VIRTUAL_HOST` being set to the same "quickstatements.local.gd" address.